warp10/src/main/java/io/warp10/standalone/StandaloneAcceleratedStoreClient.java

```diff
@@ -72,11 +89,29 @@ protected Boolean initialValue() {
     }
   };
 
-  private static final AtomicBoolean instantiated = new AtomicBoolean(false);
-  
   public StandaloneAcceleratedStoreClient(DirectoryClient dir, StoreClient persistentStore) {
+        
+    if (null != instance) {
+      throw new RuntimeException(StandaloneAcceleratedStoreClient.class.getName() + " can only be instantiated once.");
+    }
+            
+    //
+    // Force accelerator parameters to be replicated on inmemory ones and clear other in memory params
+    //
+    
+    if (null == WarpConfig.getProperty(Configuration.ACCELERATOR_CHUNK_COUNT)
+        || null == WarpConfig.getProperty(Configuration.ACCELERATOR_CHUNK_LENGTH)) {
+      throw new RuntimeException("Missing configuration key '" + Configuration.ACCELERATOR_CHUNK_COUNT + "' or '" + Configuration.ACCELERATOR_CHUNK_LENGTH + "'");
+    }
```

Comment:
The accelerator will load its in-memory cache with data from disk upon startup, having default values similar to those of in.memory.xxx would lead to massive fetch ops, so better not have default values and force the user to choose wisely.